deploy/templates/marlowe-contract.yaml

```diff
@@ -0,0 +1,75 @@
+{{- range .Values.networks }}
+---
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: marlowe-contract-{{ . }}
+  namespace: marlowe-staging
+  annotations:
+    app.oam.dev/publishVersion: {{ $.Chart.AppVersion }}
+spec:
+  components:
+    - name: marlowe-contract-{{ . }}
+      type: webservice
+      properties:
+        env:
+        - name: HOST
+          value: 0.0.0.0
+        - name: PORT
+          value: "3727"
+        - name: QUERY_PORT
+          value: "3728"
+        - name: TRANSFER_PORT
+          value: "3729"
+        - name: STORE_DIR
+          value: /store
+        - name: HTTP_PORT
+          value: "3787"
+        - name: OTEL_EXPORTER_OTLP_ENDPOINT
+          value: http://grafana-agent.grafana-agent:4318
+        - name: OTEL_SERVICE_NAME
+          value: marlowe-contract-{{ . }}
+        cpu: "0.5"
+        image: {{ $.Values.images.repo }}/{{ $.Values.images.org }}/marlowe-contract:{{ $.Values.images.tag }}
+        imagePullPolicy: Always
+        memory: 1024Mi
+        ports:
+        - expose: true
+          port: 3727
+          protocol: TCP
+        - expose: true
+          port: 3728
+          protocol: TCP
+        - expose: true
+          port: 3787
+          protocol: TCP
+      traits:
+      - type: storage
+        properties:
+          pvc:
+          - name: store-dir-{{ . }}
+            mountPath: /store
+            storageClassName: ebs-sc
+            resources:
+              requests:
+                storage: 50Gi
```

Comment:
This could run out (even merkleized, some contracts can be very large). Until we implement garbage collection, I'd suggest bumping this to at least 200Gi